integrations/postgresql/src/test/kotlin/xef/PGVectorStoreSpec.kt

```diff
@@ -63,6 +69,10 @@ class PGVectorStoreSpec :
 
     "createCollection should create collection" { pg.createCollection() }
 
+    "addTexts should not fail now that we created the collection" {
+      pg.addTexts(listOf("foo", "bar"))
```

Comment:
texts were not added, so all tests after `"createCollection should create collection"` that used embeddings were failing. This was needed!